test/index.js

```diff
@@ -0,0 +1,17 @@
+var test = require('tape'),
+  cp = require('child_process')
+
+test('node version', function (t) {
+  t.plan(2)
+  child = cp.exec('node main',
+  function (error, stdout, stderr) {
+    console.log('stdout: ' + stdout)
+    console.log('stderr: ' + stderr)
+    t.equal(stderr, '')
+    if (error !== null) {
+      console.log('exec error: ' + error)
+    }
+    stdout = stdout.replace('\n','')
+    t.equal(stdout, 'scott.wilkerson@hopeli.me')
+  })
+})
```

Comment:
these tests are pretty hard to read. similar comments to other labs - but students don't know what a child process is or how to interpret `cp.exec`....

is there a way to simplify the tests by changing the work -  to have them define functions that return specific data